rinex/src/observation/record.rs

```diff
@@ -1461,13 +1441,13 @@ fn dual_freq_combination(
                 };
 
                 let value = match combination {
-                    Combination::NarrowLane => alpha * beta * (v_i + gamma * v_j),
-                    _ => alpha * beta * (v_i - gamma * v_j),
+                    Combination::NarrowLane => alpha * (beta * v_i + gamma * v_j),
```

Comment:
This is intended, right?

Reply:
Correct, the implemented equation is simplified